kv/raftstore/peer.go

```diff
@@ -343,8 +343,13 @@ func (p *peer) Term() uint64 {
 }
 
 func (p *peer) HeartbeatScheduler(ch chan<- worker.Task) {
+	clonedRegion := new(metapb.Region)
+	err := util.CloneMsg(p.Region(), clonedRegion)
+	if err != nil {
+		return
+	}
 	ch <- &runner.SchedulerRegionHeartbeatTask{
-		Region:          p.Region(),
+		Region:          clonedRegion,
 		Peer:            p.Meta,
```

Comment:
not sure is it okay？please check it

Reply:
how could I check it, should I write test for scheduler.go?
it wroks very well in here

Reply:
I mean the `Peer` is also a pointer, please check the code to see if there is any problem like `Region`. No need to write a test.

If no problem, I will merge this PR.

Reply:
it's ok cause a `Peer`'s meta will not change once created. `PendingPeers` only contains meta, too.